src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -1 +1 @@
-ai.eto.rikai.RikaiRelationProvider
\ No newline at end of file
+ai.eto.rikai.RikaiDataSourceV2
```

Comment:
why V2 :) we dont have V1 yet :p

Reply:
See https://github.com/apache/spark/blob/master/sql/core/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

Because it is based on Data Source API V2